app/services/fast_track/hypertension_pdf_generator.rb

```diff
@@ -2,31 +2,43 @@
 
 module FastTrack
   class HypertensionPdfGenerator
-    attr_accessor :patient, :bp_data, :medications
+    attr_accessor :patient, :blood_pressure_data, :medications, :pdf
 
-    def initialize(patient, bp_data, medications)
+    def initialize(patient, blood_pressure_data, medications)
+      @pdf = Prawn::Document.new
       @patient = patient
```

Comment:
I think this `patient` object is a name hash right? Would it help to rename it something like `patient_name_hash`?